cobc/pplex.l

```diff
@@ -1817,14 +1817,14 @@ start:
 			line_overflow = 0;
 		}
 
-		memset (buff, ' ', (size_t)6);
-		/* Note we allow directive lines to start at column 7 */
-		bp = &buff[6];
+		memset (buff, ' ', (size_t)(cb_indicator_column - 1));
+		/* Note we allow directive lines to start in the indicator column */
+		bp = &buff[(cb_indicator_column - 1)];
```

Comment:
style: please always use the same as in line 1907 (in brackets without the parenthesis)